cost/google/cud_recommendations/google_committed_use_discount_recommendations.pt

```diff
@@ -34,13 +35,6 @@ parameter "param_regions" do
   description "Location to check, this is region names"
 end
 
-parameter "param_automatic_action" do
```

Comment:
Should this PR include README changes as well? 

i.e., If this has been removed from the template, shouldn't this also be removed from the Input Parameters listed in the README?

Reply:
this parameter isn't reflected in the readme.